bats/scripts/rename-logs.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+
+# GitHub restricts artifact filenames:
+
+# Invalid characters include: Double quote ", Colon :, Less than <,
+# Greater than >, Vertical bar |, Asterisk *, Question mark ?, Carriage
+# return \r, Line feed \n
+#
+# The following characters are not allowed in files that are uploaded
+# due to limitations with certain file systems such as NTFS. To maintain
+# file system agnostic behavior, these characters are intentionally not
+# allowed to prevent potential problems with downloads on different file
+# systems.
+
+if [[ -z ${1:-} ]]; then
+    echo "usage: $0 LOGDIR"
+    exit 1
+fi
+
+# Find all files and put the names into the FILES array.
+# We don't rename inside the loop to make sure the find command has
+# finished before we modify any directories it is iterating over.
```

Comment:
Also, we don't `mapfile` because bash 3.

Reply:
Yes